src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -49,15 +50,22 @@
     <SupplementalPayloadDir Condition="'$(SupplementalPayloadDir)' == ''">$(TestWorkingDir)SupplementalPayload/</SupplementalPayloadDir>
     <SupplementalPayloadFilename>SupplementalPayload.zip</SupplementalPayloadFilename>
     <SupplementalPayloadFile>$(ArchivesRoot)$(SupplementalPayloadFilename)</SupplementalPayloadFile>
+    <OverwriteOnUpload Condition="'$(OverwriteOnUpload)' == ''">false</OverwriteOnUpload>
+    <TimeoutInSeconds Condition="'$(TimeoutInSeconds)' == ''">600</TimeoutInSeconds>
 
     <!-- Workaround to bring the whole Test ILC folder if required  -->
     <TestILCVersion>1.4.24208-prerelease</TestILCVersion>
-    <TestILCFolder>$(PackagesDir)Microsoft.DotNet.TestILC\$(TestILCVersion)\contentFiles\any\any\TestILC</TestILCFolder>
-
-    <TestILCZipFileName>TestILC.zip</TestILCZipFileName>
+    <TestILCFolder>$(PackagesDir)Microsoft.DotNet.TestILC\$(TestILCVersion)\contentFiles\any\any\TestILC</TestILCFolder><TestILCZipFileName>TestILC.zip</TestILCZipFileName>
     <TestILCZipFile>$(ArchivesRoot)$(TestILCZipFileName)</TestILCZipFile>
-    <OverwriteOnUpload Condition="'$(OverwriteOnUpload)' == ''">false</OverwriteOnUpload>
-    <TimeoutInSeconds Condition="'$(TimeoutInSeconds)' == ''">600</TimeoutInSeconds>
+    
+    <IsOfficial Condition="'$(OfficialBuildId)'!=''">true</IsOfficial>
+    <CurrentDate>$([System.DateTime]::Now.ToString(yyyyMMdd.mmss))</CurrentDate>
+    <HelixJobType Condition="'$(FilterToTestTFM)'=='netcore50' and !$(TestNugetRuntimeId.Contains('aot'))">test/core/uwp</HelixJobType>
+    <HelixJobType Condition="'$(FilterToTestTFM)'=='netcore50' and $(TestNugetRuntimeId.Contains('aot'))">test/core/ilc</HelixJobType>
+    <HelixJobType Condition="$(FilterToTestTFM.Contains('net46'))">test/desktop</HelixJobType>
+    <HelixJobType Condition="'$(HelixJobType)'==''">test/core</HelixJobType>
```

Comment:
These types don't match the current type expectations, so will cause reporting to stop.  Currently the "normal" tests (the ones we've been running for months) have the type "test/functional/cli" (because they are function tests that are running using the command line interface).  "test/functional/uwp", "test/functional/ilc", and "test/functional/desktop/cli" seem reasonable approximations for the others. They are all testing functionality of NetCore in various runtime configurations.

Non functional tests (like "stress", which I think also uses this file to generate it's runs) also need to be accounted for.